templates/tab_fact/tab_fact/templates.yaml

```diff
@@ -0,0 +1,128 @@
+dataset: tab_fact
+subset: tab_fact
+templates:
+  137a6f5d-fdcd-4849-ba3c-7ae572285ef9: !Template
+    id: 137a6f5d-fdcd-4849-ba3c-7ae572285ef9
+    jinja: '{% if label %}
+
+      Passage: "{{statement}}"
+
+
+      Table: "{{table_text}}"
+
+
+      Give a suitable caption for the table. Use the statement as a supporting doc.
+      |||
+
+      {{table_caption}}
+
+      {% endif %}'
+    name: tab_fact_7
+    reference: Generate Table Caption
+    task_template: false
+  1f0606bd-0453-427f-8cc5-ab996aff680e: !Template
+    id: 1f0606bd-0453-427f-8cc5-ab996aff680e
+    jinja: 'table ==> passage?
+
+
+      table:
+
+      "{{table_caption}}"
+
+
+      {{table_text}}
+
+
+      statement: "{{statement}}" |||
+
+
+      {{["No","Yes"][label]}}'
+    name: tab_fact_4
+    reference: 'Input: Table Caption, Table; Label: Yes/No (mathematical "implication"
+      symbol)'
+    task_template: true
+  33e3dbc2-3b1b-4891-8c78-2b575dd3ec35: !Template
+    id: 33e3dbc2-3b1b-4891-8c78-2b575dd3ec35
+    jinja: "Parse the following table:\n\nTable Caption: \"{{table_caption}}\"\n\n\
+      Table:\n\n{{table_text}}\n\nNote: {{\"#\"}} is the delimiter between columns;\
+      \ {{\"newline\"}} is the delimiter between rows.\n\nFrom the above table, the\
+      \ statement \"{{statement}}\" can either be entailed or refuted. Which one is\
```

Comment:
protect "entailed" and "refuted -> `{{"entailed"}} or {{"refuted"}}`

Reply:
same for tab_fact_2

Reply:
Sure.